### PR TITLE
[INLONG-7722][Manager] Check task result before returning

### LINKED ISSUE #7722: [Bug][Manager] The task status is inconsistent with the returned result
### What happened

The task status is inconsistent with the returned result.

### What you expected to happen

Check the actual state of the task before returning the result.

### How to reproduce

Suspend or restart the group.

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by haifxu**
### Prepare a Pull Request

- Fixes #7722 

### Motivation

It's no error message displayed when the task failed.

### Modifications

Check task result before returning

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupProcessService.java [hunk 1]
```diff
@@ -104,6 +104,11 @@ public WorkflowResult startProcess(String groupId, String operator) {
         groupService.updateStatus(groupId, GroupStatus.TO_BE_APPROVAL.getCode(), operator);
         ApplyGroupProcessForm form = genApplyGroupProcessForm(groupId);
         WorkflowResult result = workflowService.start(ProcessName.APPLY_GROUP_PROCESS, operator, form);
+        List<TaskResponse> tasks = result.getNewTasks();
+        if (TaskStatus.FAILED == tasks.get(tasks.size() - 1).getStatus()) {
+            throw new BusinessException(ErrorCodeEnum.WORKFLOW_DELETE_RECORD_FAILED,
```
Comment (fuweng11) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupProcessService.java:109]: `WORKFLOW_START_RECORD_FAILED
`

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupProcessService.java [hunk 2]
```diff
@@ -146,6 +151,11 @@ public WorkflowResult suspendProcess(String groupId, String operator) {
         InlongGroupInfo groupInfo = groupService.get(groupId);
         GroupResourceProcessForm form = genGroupResourceProcessForm(groupInfo, GroupOperateType.SUSPEND);
         WorkflowResult result = workflowService.start(ProcessName.SUSPEND_GROUP_PROCESS, operator, form);
+        List<TaskResponse> tasks = result.getNewTasks();
+        if (TaskStatus.FAILED == tasks.get(tasks.size() - 1).getStatus()) {
+            throw new BusinessException(ErrorCodeEnum.WORKFLOW_DELETE_RECORD_FAILED,
```
Comment (fuweng11) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupProcessService.java:156]: WORKFLOW_SUPSPEND_RECORD_FAILED

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupProcessService.java [hunk 3]
```diff
@@ -184,6 +194,11 @@ public WorkflowResult restartProcess(String groupId, String operator) {
         InlongGroupInfo groupInfo = groupService.get(groupId);
         GroupResourceProcessForm form = genGroupResourceProcessForm(groupInfo, GroupOperateType.RESTART);
         WorkflowResult result = workflowService.start(ProcessName.RESTART_GROUP_PROCESS, operator, form);
+        List<TaskResponse> tasks = result.getNewTasks();
+        if (TaskStatus.FAILED == tasks.get(tasks.size() - 1).getStatus()) {
+            throw new BusinessException(ErrorCodeEnum.WORKFLOW_DELETE_RECORD_FAILED,
```
Comment (fuweng11) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupProcessService.java:199]: WORKFLOW_RESTART_RECORD_FAILED

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupProcessService.java (hunk 1 above)
Comment (haifxu) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupProcessService.java]: Done.

**Review by fuweng11 — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by vernedeng — APPROVED** [review]
LGTM

**Pull request merged**